Modules/_sre/sre.c

```diff
@@ -2974,6 +3202,7 @@ sre_clear(PyObject *module)
     Py_CLEAR(state->Pattern_Type);
     Py_CLEAR(state->Match_Type);
     Py_CLEAR(state->Scanner_Type);
+    Py_CLEAR(state->Template_Type);
 
```

Comment:
+ `Py_CLEAR(state->compile_template);`  ?

Reply:
Done. And in `sre_traverse`.